crates/ra_ide/src/diagnostics.rs

```diff
@@ -362,7 +371,12 @@ mod tests {
     }
 
     fn check_apply_diagnostic_fix(before: &str, after: &str) {
-        let (analysis, file_id) = single_file(before);
+        let (analysis, file_id) = if before.contains("//-") {
+            let (analysis, pos) = analysis_and_position(before);
```

Comment:
hacky way to allow multi-file fixtures

Reply:
This should no longer be necessary, there's no single syntax for multifile and single file fixtures.

Reply:
nice :)